.grit/patterns/python/use_flask_jsonify.md

```diff
@@ -0,0 +1,87 @@
+---
+title: use `flask.jsonify()`
+tags: [fix, best-pactice, flask]
+---
+
+`flask.jsonify()` is a `Flask` helper method which handles the correct settings for returning `JSON` from `Flask` routes
```

Comment:
It would be good to make descriptions more precise and explain a bit more. We now have auto-generated documentation from the Markdown. Ex. https://docs.grit.io/patterns/library/aware_utc

A second paragraph explaining *why* would be beneficial.

Reply:
True, agree with you, from now on, I will keep this in mind and make it more descriptive. updated it.


This is just a suggestion, we should update it and make it more precise, about writing `title, tags, description and examples`

https://github.com/getgrit/stdlib/blob/main/CONTRIBUTING.md

that will help other `contributors` also to understand what is the expectation.